packages/core-sdk/src/types/resources/ipAccount.ts

```diff
@@ -48,3 +48,16 @@ export type SetIpMetadataRequest = {
   metadataHash: Hex;
   txOptions?: Omit<TxOptions, "encodedTxDataOnly">;
 };
+
+export type TransferErc20Request = {
+  ipId: Address;
+  tokens: {
+    /** The address of the ERC20 token including WIP and standard ERC20. */
+    address: Address;
+    /** The amount of the ERC20 token to transfer. */
+    amount: bigint | string | number;
```

Comment:
lets use `TokenAmountInput` for all token amount in the future. It allow us to have a consistent input type across all functions and I don't think we need string support.